src/rulesets/Shopify.ruleset.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "Ruleset for Shopify connector",
+    "description": "Temporary ruleset for Shopify connector to allow migration to BcApps repo",
+    "includedRuleSets": [
+		{
+			"action": "Default",
+			"path": "./minorrelease.ruleset.json"
```

```suggestion
			"path": "./ruleset.json"
```
That's the ruleset we're using now. We should probably keep using this no? Just without this one rule